addons/packages/ako-operator/1.4.0/README.md

```diff
@@ -1,14 +1,106 @@
 # ako-operator
 
-This package provides NSX Advanced Load Balancer using ako-operator.
+This package manages the lifecycles, credentials, and certification of [AVI Kubernetes Operator (AKO)](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/release-1.5.2) in Cluster API based multi-cluster env.
+It allows per-cluster AKO configuration using the `AKODeploymentConfig` CR and deploys AKO in workload clusters automatically.
 
 ## Components
 
-* ako-operator
+* [Deployment](./bundle/config/upstream/akooperator/deployment.yaml) for the Load Balancer Operator (ako-operator)
+* [CRD](./bundle/config/upstream/akooperator/akodeploymentconfig.yaml) for `AKODeploymentConfig`
+* [Two default `AKODeploymentConfig` objects](./bundle/config/upstream/akooperator/akodeploymentconfig.yaml) for management and workload clusters respectively
+
+## Supported Providers
+
+The following tables shows the providers this package can work with. Other cloud provider support will be added  
+in the future.
+
+| AWS  |  Azure  | vSphere  | Docker |
+|:---:|:---:|:---:|:---:|
+|  ❌ |   ❌ | ✅  |  ❌  |
+
+## Mailing lists
+
+* Use [tkg-infrax-akita](mailto:tkg-infrax-akita@groups.vmware.com) to report security concerns to the AKO Team,
+  who uses the list to privately discuss security issues and fixes prior to disclosure.
```

Comment:
instead of this: <del>who uses the list to privately discuss security issues and fixes prior to disclosure.</del>, we can say: who is responsible for xxxx.